main.css

```diff
@@ -0,0 +1,5 @@
+@import url("files/button.css");
```

Comment:
order of loading css files should be corrected first is alway varibables,  reset,common, then you can load modules

Reply:
also all the files of css should be under css folder.